src/main/java/transforms/BooleanIsNullTransform.java

```diff
@@ -45,8 +45,8 @@ public boolean applyCondition( Map<String, Object> row ) {
         if ( !( row.containsKey( column ) ) ) {
             throw new IllegalStateException( String.format( "The column %s is not found.", column ) );
         }
-      
-        return row.get( column ) == null ;
+
+        return row.get( column ) == null || row.get( column ).equals( "" ) ;
```

Comment:
It's a little counterintuitive that nulls and also empty strings get the same treatment given the class name -- it would probably be clearer to rename something like `IsEmptyTransform` or something like that. 

Also -- I assume you want to treat `""` as empty but not `" "`? If you do, you should probably just use `StringUtils.isBlank( row.get( column ))`;

Reply:
True, IsEmptyTransform would be clearer. BooleanIsNullCondition was already checking for blanks (which is what we need it for), so I just decided to be consistent with that.

Reply:
update: now using isBlank for this. As far as renaming, I'd like to check in with Joke first - I believe she has her own plans in the works for restructuring transforms/conditions.